server/models/Profile.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose");
+const userSchema = require("./User");
+const email = User.find_by_id({});
```

Comment:
What is the purpose of this? Please remove or comment the code with a description :)

Reply:
Done